src/components/ceremony/ceremony.service.ts

```diff
@@ -29,13 +28,9 @@ export class CeremonyService {
 
   async create(input: CreateCeremony): Promise<ID> {
     try {
-      const result = await this.ceremonyRepo.create(input);
+      const { node } = await this.ceremonyRepo.create(input);
```

Comment:
Rename for consistency
```suggestion
      const { node } = await this.repo.create(input);
```